resim/third_party/ros2/ros2_rmw_implementation.patch

```diff
@@ -0,0 +1,62 @@
+diff --git a/rmw_implementation/src/functions.cpp b/rmw_implementation/src/functions.cpp
+index 892eb40..337d02a 100644
+--- a/rmw_implementation/src/functions.cpp
++++ b/rmw_implementation/src/functions.cpp
+@@ -20,7 +20,6 @@
+ #include <stdexcept>
+ #include <string>
+ 
+-#include "ament_index_cpp/get_resources.hpp"
+ 
+ #include "rcutils/allocator.h"
+ #include "rcutils/format_string.h"
+@@ -91,39 +90,21 @@ load_library()
+       "from environment due to %s", e.what());
+     return nullptr;
+   }
+-
+   // User specified an RMW, attempt to load that one and only that one
+   if (!env_var.empty()) {
+-    return attempt_to_load_one_rmw(env_var);
++    return std::make_shared<rcpputils::SharedLibrary>(env_var.c_str());
+   }
+-
```

Comment:
This patch differs from mvukov/rules_ros2 in that it still checks the environment variable for the rmw implementation path first as the original file does.